app/src/main/java/com/yeonae/chamelezone/view/home/HomeTabFragment.kt

```diff
@@ -102,99 +98,61 @@ class HomeTabFragment :
             startActivity(intent)
         }
 
-        val myLocationHandler = Handler()
-
-        val lm =
-            context?.getSystemService(LOCATION_SERVICE) as LocationManager?
-
-        myLocationHandler.postDelayed(object : Runnable {
-            @SuppressLint("ObsoleteSdkInt")
-            override fun run() {
-
-                if (Build.VERSION.SDK_INT >= 24 &&
-                    ContextCompat.checkSelfPermission(
-                        requireContext(),
-                        Manifest.permission.ACCESS_FINE_LOCATION
-                    ) != PackageManager.PERMISSION_GRANTED
-                ) {
-                    ActivityCompat.requestPermissions(
-                        requireActivity(),
-                        arrayOf(Manifest.permission.ACCESS_FINE_LOCATION),
-                        0
-                    )
+        presenter = HomePresenter(
+            Injection.placeRepository(), Injection.memberRepository(),
+            Injection.likeRepository(),
+            this
+        )
```

Comment:
```suggestion
        presenter = HomePresenter(
            Injection.placeRepository(), 
            Injection.memberRepository(),
            Injection.likeRepository(),
            this
        )
```
개행하여 정리하는 것이 좋을 것 같아요

Reply:
수정했습니다